src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -752,6 +753,19 @@ private static async Task ProcessJobs(string hostname, string dockerHostname, Ca
                                         });
                                     }
 
+                                    if (!job.Metadata.Any(x => x.Name == "benchmarks/symbols-size"))
+                                    {
+                                        job.Metadata.Enqueue(new MeasurementMetadata
+                                        {
+                                            Source = "Host Process",
+                                            Name = "benchmarks/symbols-size",
```

Comment:
Looks like you use the same text here as on line 756. I'd just move it into a constant and reuse in both places.